### ISSUE TITLE
Wrong barplot x-labels in https://www.tensorflow.org/tutorials/audio/simple_audio

### ISSUE THREAD

**Issue by JacekPardyak** [tensorflow/tensorflow#58612]
I think that in the notebook https://www.tensorflow.org/tutorials/audio/simple_audio this line:

`plt.bar(commands, tf.nn.softmax(prediction[0]))`

should be replaced with:

`x_labels = ['down', 'go', 'left', 'no', 'right', 'stop', 'up', 'yes']`
`plt.bar(x_labels, tf.nn.softmax(prediction[0]))`

As a result, the graph will correctly show the probabilities for all classes.

**Comment by JacekPardyak** [comment]
Line: 
`plt.bar(np.sort(commands), tf.nn.softmax(prediction[0]))`
instead of:
`plt.bar(commands, tf.nn.softmax(prediction[0]))`
will work too.

**Comment by mohantym** [comment]
Hi @JacekPardyak !
Thanks for sharing your observation on this bug. 

It is correct that probabilities for others is not visible in above output ( looked like a image rendering issue initially) 
Upon above fact, I investigated on the sequence shared by you, the present command and the one mentioned in confusion matrix. They are showing same results in output.

So I think It would be good to change the documentation  with sequence mentioned in [dataset](https://www.tensorflow.org/tutorials/audio/simple_audio#import_the_mini_speech_commands_dataset) mentions below sequence . 

Ref :
1. [output in webpage](https://user-images.githubusercontent.com/86464649/202963204-5e9c047b-da2d-4393-8de4-76ead5bcd0e7.png)

2. [output in gist](https://user-images.githubusercontent.com/86464649/202963500-c2a4aa08-db2e-4e66-8772-51a5c45eedb8.png)

3. [test_gist]( https://colab.sandbox.google.com/gist/mohantym/d1a0a3bff923c8d948ac46b577490e40/simple_audio.ipynb#scrollTo=zRxauKMdhofU)

Thank you!

**Comment by 8bitmp3** [comment]
@JacekPardyak On it

**Comment by 8bitmp3** [comment]
PR https://github.com/tensorflow/docs/pull/2152 approved

**Comment by JacekPardyak** [comment]
Dear team,
nothing has changed for the better for me. The first graph is from https://www.tensorflow.org/tutorials/audio/simple_audio which still shows wrong x labels in the bar plot.

![CaptureIs](https://user-images.githubusercontent.com/15980055/208459852-291a09ea-a1f8-4e85-966a-66fff5b73d3d.PNG)

The second graph I obtained myself after modifying the code:

![CaptureShouldBe](https://user-images.githubusercontent.com/15980055/208459858-a11a86fc-5515-4662-9d2a-ac611e3bb955.PNG)

The modification is to sort the numpy array. Namely replacing the first line below with the second line below:

`plt.bar(commands, tf.nn.softmax(prediction[0]))`
`plt.bar(np.sort(commands), tf.nn.softmax(prediction[0]))`

in the code on the previously mentioned page.

**Comment by mohantym** [comment]
Hi @JacekPardyak !

np.sort(commands) will only put the commands in their phonetical sequence (d, g , l ..etc ) not as per dataset sequence.
As per my earlier investigation in the tutorial , The sequence in dataset upon which model was trained was as below.
` no, yes, down, go, left, up, right, and stop:`

Wont be reasonable to go with above sequence instead of np.sort or setting the sequence which you suggested earlier.

Thank you!
